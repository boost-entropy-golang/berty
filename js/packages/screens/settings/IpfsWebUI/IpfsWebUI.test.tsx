import { renderScreen } from '@berty/utils/testing/renderScreen.test'

import { IpfsWebUI } from './IpfsWebUI'

test('Settings.IpfsWebUI renders correctly', () => {
	const { toJSON } = renderScreen('Settings.IpfsWebUI', IpfsWebUI)
	expect(toJSON()).toMatchSnapshot()
})
