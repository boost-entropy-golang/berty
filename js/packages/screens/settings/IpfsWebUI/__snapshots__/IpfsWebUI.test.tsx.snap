// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings.IpfsWebUI renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      appearance="default"
      level="1"
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
          },
          Object {
            "backgroundColor": "#FFFFFF",
            "flex": 1,
          },
        ]
      }
    >
      <RCTScrollView
        bounces={false}
      >
        <View>
          <View
            style={
              Array [
                Object {
                  "padding": 16,
                },
                Object {
                  "flex": 1,
                },
                Object {
                  "marginBottom": 9,
                },
              ]
            }
          >
            <View
              style={
                Array [
                  Object {
                    "flex": 1,
                    "overflow": "hidden",
                  },
                  undefined,
                ]
              }
            >
              <RNCWebView
                cacheEnabled={true}
                injectedJavaScriptBeforeContentLoadedForMainFrameOnly={true}
                injectedJavaScriptForMainFrameOnly={true}
                javaScriptEnabled={true}
                messagingEnabled={false}
                onContentProcessDidTerminate={[Function]}
                onHttpError={[Function]}
                onLoadingError={[Function]}
                onLoadingFinish={[Function]}
                onLoadingProgress={[Function]}
                onLoadingStart={[Function]}
                onMessage={[Function]}
                onShouldStartLoadWithRequest={[Function]}
                source={
                  Object {
                    "uri": "http://127.0.0.1:3000",
                  }
                }
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "overflow": "hidden",
                    },
                    Object {
                      "backgroundColor": "#ffffff",
                    },
                    Array [
                      Object {
                        "height": 1000,
                      },
                    ],
                  ]
                }
                useSharedProcessPool={true}
              />
            </View>
          </View>
        </View>
      </RCTScrollView>
    </View>
  </View>
</RNCSafeAreaProvider>
`;
